### PR TITLE
Update changelog 1.1

### PR DESCRIPTION
**Pull request by SchmErik**
Update the change log in preparation for the 1.1 release. For this update, I've tried including links to pull requests and made an attempt at making the changelog more accessible.

### CODE REVIEW

**Comment by SchmErik** [comment]
I've tagged several people here.
@santiagocampos12 - for visibility. Let me know something isn't clear or if you have any questions!
@nuke-web3, @sashaaldrick, @pdg744, @ssh-nkar for visibility.
@tzerrell - could you review my blub about the zk-ness change?
@shkoo - I'm not sure how to phrase some of the changes you've made since 1.0. Maybe we can chat offline?
@flaub - general review would be great but also could check my claim on the CUDA changes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,33 @@
 
 ## Next (upcoming release)
 
-TBD
+### 🔥 Performance Improvements
+* CUDA performance improvements. This results in about a 20-30% performance improvement. https://github.com/risc0/risc0/pull/2054
```
Comment (nuke-web3) [inline CHANGELOG.md:6]: LOVE the links to pull requests to look for more details

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,33 @@
...
+* Enable GPU proving by default on MacOS. Previously, we have allowed the user to prove on CPU or GPU. This improves user experience by shipping a GPU-enabled r0vm server by default. https://github.com/risc0/risc0/pull/2092
+* Added an example that showcases how the `c-kzg` crate can be run in the zkVM. https://github.com/risc0/risc0/pull/2045
+* Added an example that uses C as the guest code. This demonstrates how a user can use C to write an entire guest program without rust. https://github.com/risc0/risc0/pull/1898
+* Added a simple prover service example. This is a demonstration of how the public API can be used to build a simple proving service. https://github.com/risc0/risc0/pull/2113
+* Added the ability to serialize Receipts using the borsh serialization crate. https://github.com/risc0/risc0/pull/2184
+
+### 🚨 Breaking Changes
+
+* Due to the zero-knowledge change above, receipts generated by zkVM 1.0.x cannot be verified with IDs from 1.1.x and verifiers using control IDs from 1.0.x are not compatible with 1.1.x.
```
Comment (nuke-web3) [inline CHANGELOG.md:31]: good to know - is there any actions we can suggest and/or examples of some diff that all (or most) people will need to enact on their projects using us? 

Basically we have "what" but are missing the "why" and "how (to deal with this)"

Comment (SchmErik) [reply CHANGELOG.md:31]: done!

Comment (SchmErik) [reply CHANGELOG.md:31]: Thanks for this feedback, I appreciate it!

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,33 @@
...
 ## Next (upcoming release)
 
-TBD
+### 🔥 Performance Improvements
+* CUDA performance improvements. This results in about a 20-30% performance improvement. https://github.com/risc0/risc0/pull/2054
+* Add recursion preflight support for checked bytes/bigint https://github.com/risc0/risc0/pull/1973
+
+### 🛠 Fixes
+
+* Improve zero-knowledge property of the prover by increasing the amount of noise. We have used techniques described in this [paper](https://eprint.iacr.org/2024/1037.pdf) to strengthen the ZK property of the prover. This change results in a new set of control IDs. See the breaking changes section to learn more about this minor breaking change. https://github.com/risc0/risc0/pull/1970
```
Comment (tzerrell) [inline CHANGELOG.md:11]: Maybe also link to our ZK section of our docs?
```suggestion
* Improve zero-knowledge property of the prover by increasing the amount of noise. We have used techniques described in this [paper](https://eprint.iacr.org/2024/1037.pdf) to strengthen the ZK property of the prover. See [these docs](https://dev.risczero.com/api/security-model#zero-knowledge-proving) for more information about zero knowledge and our proving system. This change results in a new set of control IDs. See the breaking changes section to learn more about this minor breaking change. https://github.com/risc0/risc0/pull/1970
```

**Review by tzerrell — APPROVED** [review]
Looks good; I made a suggested addition inline but I'm not sure it's necessary.

File: CHANGELOG.md (hunk 3 above)
Comment (SchmErik) [reply CHANGELOG.md]: @tzerrell this is excellent. Thank you!

**Review by flaub — APPROVED** [review]

**Pull request merged**